src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -962,7 +962,22 @@ function getWarningMessage() {
  * @param textAreaId - Id of text area for which char are to be counted
  * @param charCountId - Id of Label to display length of text area
  */
-function updateTextQuestionCharCount(textAreaId, charCountId) {
-    var cs = $('#' + textAreaId).val().length;
-    $('#' + charCountId).text(cs);
+function updateTextQuestionCharCount(textAreaId, charCountId, recommendedLength) {
+    var cs = $('#' + textAreaId).val().trim().split(/\s+/).length;
```

Comment:
As `$('#' + textAreaId)` and `$('#' + charCountId)` are used multiple times, it would be good to extract them into variables